moveit_planners/test_configs/prbt_moveit_config/config/kinematics.yaml

```diff
@@ -4,7 +4,7 @@ manipulator:
   #kinematics_solver_timeout: 0.05
   #link_prefix: prbt_
   #prefix: prbt_
-  kinematics_solver: prbt_manipulator/IKFastKinematicsPlugin
-  kinematics_solver_search_resolution: 0.005
+  kinematics_solver: ["prbt_manipulator/IKFastKinematicsPlugin"]
+  kinematics_solver_search_resolution: [0.005]
```

Comment:
I don't think you want to make these array types. For the first one the quotes are optional as the default type with characters in it is string when writing yaml.

Reply:
I thought of making it array type since in code, it takes those parameters as string and splits for multiple solvers using white space. 
Even the search resolution is declared as a string if it needs to be different values for different solvers - https://github.com/ros-planning/moveit2/blob/main/moveit_ros/planning/kinematics_plugin_loader/src/kinematics_plugin_loader.cpp#L357-L372

Reply:
To me this is an unnecessary extension point. @henningkayser what do you think of changing this so it only creates one solver / group.

Reply:
If you make this an array (which is a much more correct way to specify multiple things in a parameter than a string with a space) it will break existing configs. I think I'd rather break configs that were using the space hack.